google/cloud/spanner_v1/database.py

```diff
@@ -694,7 +698,7 @@ def execute_partitioned_dml(
             )
 
         def execute_pdml():
-            with SessionCheckout(self._pool) as session:
+            def do_execute_pdml(session):
                 txn = api.begin_transaction(
```

Comment:
wrap this in trace_call to create a new span `CloudSpannerOperation.BeginTransaction`?
Ideally, the traces for pdml will look like,
+ CloudSpanner.execute_partitioned_dml
      + Some session related span if there are any
      +. CloudSpannerOperation.BeginTransaction
      + CloudSpannerOperation.ExecuteStreamingSql